mmeval/metrics/gradient_error.py

```diff
@@ -0,0 +1,176 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import cv2
+import numpy as np
+from typing import Dict, List, Sequence
+
+from mmeval.core import BaseMetric
+
+
+def gaussian(x, sigma):
+    """Gaussian function.
+
+    Args:
+        x (array_like): The independent variable.
+        sigma (float): Standard deviation of the gaussian function.
+
+    Return:
+        np.ndarray or scalar: Gaussian value of `x`.
+    """
+    return np.exp(-x**2 / (2 * sigma**2)) / (sigma * np.sqrt(2 * np.pi))
+
+
+def dgaussian(x, sigma):
+    """Gradient of gaussian.
+
+    Args:
+        x (array_like): The independent variable.
+        sigma (float): Standard deviation of the gaussian function.
+
+    Return:
+        np.ndarray or scalar: Gradient of gaussian of `x`.
+    """
+    return -x * gaussian(x, sigma) / sigma**2
+
+
+def gauss_filter(sigma, epsilon=1e-2):
+    """Gradient of gaussian.
+
```

Comment:
It looks a bit weird: the function name is `gauss_filter` while the summary line is `Gradient of gaussian`.

Reply:
Yes, it has been modified in the latest update.